frontend/app/src/components/account-management/DockerWarning.vue

```diff
@@ -9,20 +10,18 @@ const proceed = () => {
 </script>
 
 <template>
-  <div class="pa-6 pb-2">
-    <VAlert type="warning" text prominent class="pl-8">
-      <div class="ml-4">
-        <i18n path="docker_warning.title" tag="div">
-          <BaseExternalLink
-            :text="t('docker_warning.documentation')"
-            :href="usageGuideUrl + '#docker'"
-          />
-        </i18n>
-      </div>
-    </VAlert>
+  <div v-if="!dockerRiskAccepted && isDocker" class="max-w-[41.25rem] mx-auto">
```

Comment:
hmmm wouldn't it better to have the boolean condition check where we use the `DockerWarning` component to avoid instantiating the component at all? 

Is there something that doesn't allow us to do this?

Reply:
Yeah I wanted to arrange it, so I don't need to import the `dockerRiskAccepted` and the `isDocker` on the parent component

Reply:
I move it anyway